### PR TITLE
BugFix Enom::Domain.all where there is only one domain returned

### PR DESCRIPTION
**Pull request by ceccec**
Bug fix where there is only one domain registered

### CODE REVIEW

File: lib/enom.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
 require File.expand_path('../enom/client',  __FILE__)
 require File.expand_path('../enom/contact_info',   __FILE__)
 require File.expand_path('../enom/domain',  __FILE__)
+require File.expand_path('../enom/certificate',   __FILE__)
```
Comment (bensie) [inline lib/enom.rb]: Can you move the certificate stuff to a separate PR?

File: lib/enom/domain.rb [hunk 2]
```diff
@@ -96,6 +96,7 @@ def self.check_multiple_tlds(sld, tlds = "*")
     # Find and return all domains in the account
     def self.all(options = {})
       response = Client.request("Command" => "GetAllDomains")["interface_response"]["GetAllDomains"]["DomainDetail"]
+      response = [response] unless response.is_a?(Array)
```
Comment (bensie) [inline lib/enom/domain.rb]: So if there is only one domain registered, the API does not return an array despite the fact that it's a collection endpoint?

If that's the case, I'd prefer to use `Array[response].flatten` instead. Any chance you could include a test for this as well?

Comment (ceccec) [inline lib/enom/domain.rb]: Noted and done. I will send a pull request after I move the SSL to a different project as you asked.

File: lib/enom.rb (hunk 1 above)
Comment (ceccec) [inline lib/enom.rb]: Ok.
